ui/packages/catalog-ui-search/src/main/webapp/component/query-editor/query-editor.view.js

```diff
@@ -140,6 +140,9 @@ module.exports = Marionette.LayoutView.extend({
             case 'custom':
                 this.showCustom();
                 break;
+            case 'result':
```

Comment:
❓ I'm not sure that changes in this file actually affect anything related to the `pop up the lightbox and allow the user to edit their result form` behavior